lib/components/viewers/nearby/stop.tsx

```diff
@@ -96,35 +111,90 @@ const Stop = ({
         homeTimezone={homeTimezone}
         key={index}
         pattern={st.pattern}
-        roundedTop={false}
         route={st.pattern.route}
         stopTimes={sortedStopTimes}
       />
     )
   })
+}
+
+const Stop = ({
+  fromToSlot,
+  homeTimezone,
+  nearbyViewConfig,
+  routeSortComparator,
+  stopData
+}: Props): JSX.Element => {
+  const patternArray = patternArrayforStops(stopData, routeSortComparator)
+  const patternRows = renderPatternRows(
+    stopData,
+    patternArray,
+    homeTimezone,
+    nearbyViewConfig
+  )
+
   const inHomeTimezone = homeTimezone && homeTimezone === getUserTimezone()
   const timezoneWarning = !inHomeTimezone && (
     <StyledAlert>
       <TimezoneWarning homeTimezone={homeTimezone} />
     </StyledAlert>
   )
 
+  const isParentStop = stopData.stops?.length && stopData.stops?.length > 0
+
   return (
     <Card>
       <StopCardHeader
         actionIcon={Calendar}
         // Remove entityId URL parameter when leaving nearby view.
         actionParams={{ entityId: undefined }}
-        actionPath={`/schedule/${stopData.gtfsId}`}
+        actionPath={!isParentStop ? `/schedule/${stopData.gtfsId}` : undefined}
         actionText={
-          <FormattedMessage id="components.StopViewer.viewSchedule" />
+          !isParentStop ? (
+            <FormattedMessage id="components.StopViewer.viewSchedule" />
+          ) : undefined
         }
         fromToSlot={fromToSlot}
         stopData={stopData}
       />
       <div>
         <div>{timezoneWarning}</div>
```

Comment:
Could we add a `margin-bottom: 1rem` on the departure timezone notice? I think it could help the overall spacing (not just for stations) because it matches the margin between it and the from/to picker.
| Station | Stops|
|--------|-------|
|![image](https://github.com/user-attachments/assets/84cf7ab4-988e-49dc-9496-66dabed6fb75)|![image](https://github.com/user-attachments/assets/975974fc-1ce0-4d72-a9a5-29adc0960362)|

Reply:
I feel like that is too much whitespace tbh, not a fan. Especially because it's repeated on each card, it is a lot of extra space for something that isn't that important.